apis/python/tests/test_sparse_nd_array.py

```diff
@@ -469,7 +431,7 @@ def test_csr_csc_2d_read(tmp_path, shape):
     # We want to test read_format == "none_of_the_above", to ensure it throws NotImplementedError,
     # but that can't be gotten past typeguard.
     "read_format",
-    ["table", "coo", "csr", "csc"],
```

Comment:
please do not delete COO tensor support (I'll stop noting this in further comments, but there are other places it is deleted)